extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -336,8 +353,58 @@ export class ConsentUI {
     classList.remove(consentUiClasses.in);
     this.isIframeVisible_ = false;
     removeElement(dev().assertElement(this.ui_));
+
+    // TODO (torch2424): Hide mask if publisher provides the option
+    // this.hideMaskElement_();
+  }
+
+  /**
+   * Disables scrolling on the document
+   * @private
+   */
+  disableScroll_() {
+    if (this.scrollEnabled_) {
+      this.viewport_.enterOverlayMode();
+      this.scrollEnabled_ = false;
+    }
+  }
+
+  /**
+   * Disables scrolling on the document
+   * @private
+   */
+  enableScroll_() {
+    if (!this.scrollEnabled_) {
+      this.viewport_.leaveOverlayMode();
+      this.scrollEnabled_ = true;
+    }
   }
 
+  /**
+   * Shows the mask element
+   * @private
+   */
+  showMaskElement_() {
+    if (!this.maskElement_) {
+      const mask = this.win_.document.createElement('div');
+      mask.classList.add('i-amphtml-consent-ui-mask');
```

Comment:
Let's add the classList to consentUiClasses object. Since this function is not used now, I am fine with us doing it later : )

Reply:
Sounds good! Will mention this in the follow up PR 😄